implementations/rust/ockam/ockam_core/src/message.rs

```diff
@@ -43,25 +43,36 @@ impl From<serde_bare::Error> for crate::Error {
 /// without requiring changes in the user message types.
 pub struct Routed<M: Message> {
     inner: M,
-    route: Route,
+    return_: Route,
+    onward: Route,
```

Comment:
could we make these fields `return_route` and `onward_route`?

Reply:
I think we should do that in a separate PR. @spacekookie what do you think?

Reply:
Different PR works 👍

Reply:
Sorry, I didn't see the followup messages before doing a push. I reverted the commit. Will open a PR in a sec